fe/fe-core/src/main/java/org/apache/doris/qe/StmtExecutor.java

```diff
@@ -1836,6 +1841,41 @@ private void handleExplainStmt(String result) throws IOException {
         context.getState().setEof();
     }
 
+    private void handleLoadStmt() {
+        try {
+            LoadStmt loadStmt = (LoadStmt) parsedStmt;
+            EtlJobType jobType = loadStmt.getEtlJobType();
+            if (jobType == EtlJobType.UNKNOWN) {
+                throw new DdlException("Unknown load job type");
+            }
+            if (jobType == EtlJobType.HADOOP) {
+                throw new DdlException("Load job by hadoop cluster is disabled."
+                        + " Try using broker load. See 'help broker load;'");
+            }
+            LoadManager loadManager = context.getEnv().getLoadManager();
+            if (jobType == EtlJobType.LOCAL_FILE) {
+                if (!context.getCapability().isClientLocalFile()) {
+                    throw new DdlException("Doris server does not support load local file from mysql client.");
```

Comment:
The error msg is confusing.
The `if` says `not a client local file`, but err msg says "not support local file from client"?

Reply:
use a `1148` error code of these case